justfile

```
@@ -32,6 +32,10 @@ clean:
 build *args="//...":
   bazel build {{args}}
 
+# example: just watch run -- serve $(pwd)/samples/helloworld/config.capnp
+run *args="-- --help":
+  bazel run //src/workerd/server:workerd {{args}}
```

Might be nice to bake the `--watch --verbose --experimental` flags into this to cut down on boilerplate
```suggestion
  bazel run //src/workerd/server:workerd {{args}} --watch --verbose --experimental`
```

---

@mikea I'd definitely apply James' suggestion